store/views.py

```diff
@@ -73,10 +96,51 @@ def loanBookView(request):
 You need to accept the book id as argument from a post request.
 You additionally need to complete the returnBook function in the loaned_books.html file
 to make this feature complete
-''' 
+'''
 @csrf_exempt
 @login_required
+@require_http_methods(["POST"])
 def returnBookView(request):
-    pass
+    response_data = {
+        'message': None,
+    }
+    book_id=request.POST['bid']
+    try:
+        book=BookCopy.objects.get(pk=book_id)
+        book.status=True
+        book.borrower=None
+        book.borrow_date=None
+        book.save()
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.